src/cmake/externalpackages.cmake

```diff
@@ -107,6 +107,10 @@ else ()
     find_package (Boost 1.53 REQUIRED
                   COMPONENTS ${Boost_COMPONENTS})
 
+    if (CMAKE_GENTOO_BUILD OR CMAKE_BUILD_TYPE STREQUAL Gentoo)
+        # Gentoo's patches to CMake make sure that we will always find exactly the version of Python required by Gentoo
+        find_package (Boost 1.42 COMPONENTS ${Boost_COMPONENTS} python)
```

Comment:
I'm a little worried about the designation of boost 1.42 here when we've advertised the current minimum as 1.53. Was this on purpose, or just a cut-and-paste from someplace else where we have a version mismatch?

Reply:
This patch was originally against version 1.7.13, but it applied cleanly against current master. Thus I did not notice that the requirement on Boost changed in the meantime. I fixed my patch to also require Boost 1.53.